File: app/src/main/java/di/penopllast/com/inforemover/MainActivity.java
package di.penopllast.com.inforemover;

import android.content.Intent;
import android.content.pm.PackageManager;
import android.content.pm.ResolveInfo;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;

import java.util.ArrayList;
import java.util.List;

import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.OnClick;
import di.penopllast.com.inforemover.entities.ApplicationInfo;

public class MainActivity extends AppCompatActivity {

    private static final String TAG = "MainActivity";

    @BindView(R.id.app_list_button) Button appListButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ButterKnife.bind(this);
    }

    @Override
    protected void onResume() {
        super.onResume();
        new Handler().postDelayed(appListButton::performClick, 100);
    }

    @OnClick(R.id.app_list_button)
    protected void appListButtonPressed() {
        startActivity(new Intent(this, AppListActivity.class));
    }
}

File: app/src/main/java/di/penopllast/com/inforemover/MainActivity.kt
package di.penopllast.com.inforemover

import android.content.Intent
import android.os.Handler
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    override fun onResume() {
        super.onResume()
        app_list_button.setOnClickListener {
            startActivity(Intent(this, AppListActivity::class.java))
        }
        Handler().postDelayed({ app_list_button.performClick() }, 100)
    }

    companion object {
        private val TAG = "MainActivity"
    }
}
